backends-velox/src/test/scala/org/apache/gluten/execution/FallbackSuite.scala

```diff
@@ -71,6 +71,23 @@ class FallbackSuite extends VeloxWholeStageTransformerSuite with AdaptiveSparkPl
     collect(plan) { case v: VeloxColumnarToRowExec => v }.size
   }
 
+  test("fallback with shuffle manager") {
+    withSQLConf(GlutenConfig.COLUMNAR_SHUFFLE_ENABLED.key -> "false") {
+      runQueryAndCompare("select c1, count(*) from tmp1 group by c1") {
+        df =>
+          val plan = df.queryExecution.executedPlan
+          val columnarShuffle = find(plan) {
+            case _: ColumnarShuffledJoin => true
```

Comment:
We should check the number of`ColumnarShuffleExchangeExec` and `ShuffleExchangeExec`. You can use repartition hint to construct the sql https://spark.apache.org/docs/latest/sql-ref-syntax-qry-select-hints.html#partitioning-hints

e.g. `select /*+ REPARTITION(3, c1) */ * FROM tmp1;`

Then the check should be like:

```
assert(collectColumnarShuffleExchange(plan) == 0)
assert(collectShuffleExchange(plan) == 1)
```

Reply:
For the current query this is the plan
```
AdaptiveSparkPlan isFinalPlan=true
+- == Final Plan ==
   VeloxColumnarToRowExec
   +- ^(6) HashAggregateTransformer(keys=[c1#14], functions=[count(1)], output=[c1#14, count(1)#26L])
      +- ^(6) InputIteratorTransformer[c1#14, count#30L]
         +- ^(6) InputAdapter
            +- ^(6) RowToVeloxColumnar
               +- ^(6) AQEShuffleRead coalesced
                  +- ^(6) ShuffleQueryStage 0
                     +- Exchange hashpartitioning(c1#14, 5), ENSURE_REQUIREMENTS, [plan_id=281]
                        +- VeloxColumnarToRowExec
                           +- ^(5) HashAggregateTransformer(keys=[c1#14], functions=[partial_count(1)], output=[c1#14, count#30L])
                              +- ^(5) NativeFileScan parquet spark_catalog.default.tmp11111[c1#14] Batched: true, DataFilters: [], Format: Parquet, Location: InMemoryFileIndex(1 paths)[file:/home/anvicto/src/Gluten/spark-warehouse/org.apache.gluten.execut..., PartitionFilters: [], PushedFilters: [], ReadSchema: struct<c1:int>
```
Can I retain it?

Reply:
It's ok to me. Thanks.

Reply:
I have raised one small PR here https://github.com/apache/incubator-gluten/pull/6055 to enable fallback for shuffle.